examples/README.md

```diff
@@ -1,6 +1,6 @@
 # Node.js Examples
 
-Here we have put together a hand full of Node.js examples to help get you started. As always feel free to [contact us](https://lob.com/support) directly if you have any questions on implementation.
+Here we have put together a hand full of Node.js examples to help get you started. Please read through the official [API Documentation](README.md#api-documentation) to get a complete sense of what to expect from each endpoint. As always, feel free to [contact us](https://lob.com/support) directly if you have any questions on implementation.
```

Comment:
@ami I believe `../README.md#api-documentation` should work. Let me know if you need help!